core/src/main/java/com/ritense/valtimo/service/CamundaProcessService.java

```diff
@@ -323,6 +337,70 @@ private void setProcessesExecutable(BpmnModelInstance bpmnModel) {
         );
     }
 
+    private void setToNullWhenServiceTaskExpressionIsEmpty(BpmnModelInstance bpmnModel) {
+        bpmnModel.getModelElementsByType(ServiceTask.class).forEach(task -> {
+            if (task.getCamundaType() == null
+                && task.getCamundaClass() == null
+                && task.getCamundaExpression() == null
+                && task.getCamundaDelegateExpression() == null) {
+                task.setCamundaExpression("${null}");
+                task.setCamundaAsyncAfter(true);
+            }
+        });
+    }
+
+    private void setToNullWhenSendTaskExpressionIsEmpty(BpmnModelInstance bpmnModel) {
+        bpmnModel.getModelElementsByType(SendTask.class).forEach(task -> {
+            if (task.getCamundaType() == null
+                && task.getCamundaClass() == null
+                && task.getCamundaExpression() == null
+                && task.getCamundaDelegateExpression() == null) {
+                task.setCamundaExpression("${null}");
+                task.setCamundaAsyncAfter(true);
+            }
+        });
+    }
+
+    private void setToCorrelateAllWhenMessageSendEventExpressionIsEmpty(BpmnModelInstance bpmnModel) {
+        Stream.of(IntermediateThrowEvent.class, EndEvent.class)
+            .flatMap(sendEventClass -> bpmnModel.getModelElementsByType(sendEventClass).stream())
+            .filter(sendEvent -> sendEvent.getId().matches("Event_[a-z0-9]{7}"))
+            .flatMap(sendEvent -> sendEvent.getChildElementsByType(MessageEventDefinition.class).stream())
+            .forEach(event -> {
+                if (event.getCamundaType() == null
+                    && event.getCamundaClass() == null
+                    && event.getCamundaExpression() == null
+                    && event.getCamundaDelegateExpression() == null) {
+                    String messageName = event.getMessage() == null ? "MY_MESSAGE" : event.getMessage().getName();
+                    event.setCamundaExpression(
+                        "${correlationService.sendMessageToAll(\"" + messageName + "\",execution)}"
+                    );
+                }
+            });
+    }
+
+    private void setToRoleUserWhenUserTaskCandidateGroupsIsEmpty(BpmnModelInstance bpmnModel) {
```

Comment:
@Klaas-Ritense why would we do this?

Reply:
@Klaas-Ritense I totally understand that, and I think that it's a great improvement for the service task implementation + expression. But that's only because we're pretty confident that this is the right default value. For timers, there is no logical default value and we shouldn't make it easier to make configuration mistakes. 

As for the candidate users: it's not a required field, why would we even set it to ROLE_USER?